service.watchedlist/lib/watchedlist/watchedlist.py

```diff
@@ -660,14 +660,19 @@ def get_watched_xbmc(self, silent):
                             break
                         if modus == 'movie':
                             name = item['title'] + ' (' + str(item['year']) + ')'
-                            res = re.compile(r'tt(\d+)').findall(item['imdbnumber'])
-                            if not res:
-                                # no imdb-number for this movie in database. Skip
-                                utils.log(u'get_watched_xbmc: Movie %s has no imdb-number in database. movieid=%d. Try rescraping' % (name, int(item['movieid'])), xbmc.LOGINFO)
-                                continue
-                            imdbId = int(res[0])
-                            if imdbId == 0:
-                                utils.log(u'get_watched_xbmc: Movie %s has no valid imdb-number in database (is Null). movieid=%d. Try rescraping' % (name, int(item['movieid'])), xbmc.LOGINFO)
+                            try:
+                                # check if movie number is in imdb-format (scraper=imdb)
+                                res = re.compile(r'tt(\d+)').findall(item['imdbnumber'])
+                                if not res:
+                                    # movie number is in themoviedb format
+                                    imdbId = int(item['imdbnumber'])
+                                else:
+                                    imdbId = int(res[0])
+                            except BaseException: 
+                                # no imdb-number or themoviedb-number for this movie in database. Skip
+                                utils.log(u'get_watched_xbmc: Movie %s has no imdb-number or themoviedb-number in database. movieid=%d. Try rescraping' % (n
```

Comment:
@SchapplM your addon is broken here, that's why addon-checker is failling. You're spreading this into 2 lines (673 + 674 and the `name` variable is broken.

Reply:
Thanks for looking at this. I somehow did not see this (and merged a PR into my addon repo without checking and testing ...)